build.gradle.kts

```diff
@@ -126,20 +127,21 @@ val copyMavenScripts = tasks.register<Copy>("copyMavenScripts") {
     group = "build"
     description = "Copies the Maven source and generated scripts to output directory."
 
-    inputs.property("project.version", appVersion)
+    val projectVersion = appVersion
```

Comment:
Note: had to copy this into a local variable in order to maintain configuration caching.

Reply:
I suggest to add this as a comment, e.g.

```
    val projectVersion = appVersion. // local variable required for configuration cache compatibility
```

Please do the same for Maven further down, too.